README.md

```diff
@@ -156,6 +156,21 @@ Copy the same steps as Local but use Live URLs for each of the endpoints, intera
 
 ---
 
+## Create new reports
+
+How to add new reports to the spreadsheet to make them available in Slack.
+
+- Open your Google Spreadsheet
+- Create a new tab
+- Give it a name, e.g. _new_report_
```

Comment:
Is it ok to have spaces in the name?

Reply:
Left an extra comment to ask for no spaces, Spaces may be allowed but haven't been tested